src/main/java/mat/client/measure/ManageMeasurePresenter.java

```diff
@@ -819,6 +880,27 @@ public void onSuccess(SaveMeasureResult result) {
         }
     }
 
+    private void handleInvalidEMeasureIdGenerator(SaveMeasureResult result) {
+        InvalidEMeasureIdGeneratorDialogBox dialogBox = new InvalidEMeasureIdGeneratorDialogBox();
+        boolean isFhir = ModelTypeHelper.isFhir(currentDetails.getMeasureModel());
+        if (currentDetails.getCQLLibraryName().equalsIgnoreCase("CMS" + currentDetails.geteMeasureId())) {
+            dialogBox.displayInvalidEMeasureIdMessage((isFhir ? "CMS ID " : "eCQM ID ")
+                    + currentDetails.geteMeasureId() + " has been taken, So your ID will be " + result.geteMeasureId() + ". Do you wish to continue?");
```

Comment:
Suggest dropping the ask, since there is only 1 option.
```suggestion
                    + currentDetails.geteMeasureId() + " has been taken, so your ID will be " + result.geteMeasureId() + ".");
```